lib/Models/Cesium.ts

```diff
@@ -842,124 +836,7 @@ export default class Cesium extends GlobeOrMap {
   }
 
   doZoomTo(target: any, flightDurationSeconds = 3.0): Promise<void> {
```

Comment:
Can you please type `target` here.

Also, can you please use `ZoomTarget` here 

https://github.com/TerriaJS/terriajs/blob/bdb0b606b1e59b82731c9afbc7c2e5d8530562f7/lib/Models/NoViewer.ts#L28-L31